zombies.js

```diff
@@ -79,6 +86,24 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+  this._pack = [];
```

Comment:
putting `_pack` and `_maxHealth` on this makes it public. Just do:
`const _pack = [];` to make it private. Same for max health. You will need to define both the `this.getPack()` and `this.getMaxHealth()` within the constructor to create function that allow you access to those variables.